panoramapublic/src/org/labkey/panoramapublic/view/search/panoramaWebSearch.jsp

```diff
@@ -376,33 +406,47 @@
+                                name: peptideSearchPanelItemId,
                                 itemId: peptideSearchPanelItemId,
                                 title: 'Peptide Search',
                                 cls: 'non-ext-search-tab-panel',
                                 layout: {type: 'hbox', align: 'left'},
                                 items : [
                                     {
                                         xtype: 'textfield',
+                                        name: 'Peptide Sequence',
                                         fieldLabel: 'Peptide Sequence',
                                         itemId: peptideSequenceItemId,
-                                        labelCls: 'labkey-form-label',
+                                        labelAlign: 'top',
                                         labelWidth: 125,
                                         listeners: {
                                             render: function (comp, eOpts) {
                                                 checkAndFillValuesFromUrl(peptideSequenceItemId, comp);
+                                                new Ext4.ToolTip({
+                                                    target : comp.getEl(),
+                                                    html: 'Enter the peptide sequence to find, or multiple sequences separated by commas. Use * to match any sequence of characters.'
+                                                });
                                             }
                                         }
                                     },
+                                    {
+                                        xtype: 'tbspacer',
+                                        width:10
+                                    },
                                     {
                                         xtype: 'checkbox',
                                         fieldLabel: 'Exact Matches Only',
+                                        name: exactPeptideMatchesItemId,
                                         itemId: exactPeptideMatchesItemId,
-                                        labelCls: 'labkey-form-label',
                                         input: true,
-                                        labelWidth: 125,
+                                        labelWidth: 130,
                                         listeners: {
                                             render: function (comp, eOpts) {
                                                 checkAndFillValuesFromUrl(exactPeptideMatchesItemId, comp);
+                                                new Ext4.ToolTip({
+                                                    target : comp.getEl(),
+                                                    html: 'If checked, the search will match the peptides exactly; if unchecked, it will match any peptide that contain the specified sequence and ignore modifications.'
```

Comment:
This seems suspect too - where are we switching between searching the modified and unmodified forms?